src/main/resources/assets/origins/lang/tr_tr.json

```diff
@@ -1,247 +1,247 @@
+  "origin.origins.feline.name": "Feline",
+  "origin.origins.feline.description": "Kedimsi görünümleriyle Felinelar, creeperların ödünü patlatır. Kedi refleksleri sayesinde her zaman dört ayak üzerine düşerler.",
 
-  "origin.origins.elytrian.name": "Elitriyan",
-  "origin.origins.elytrian.description": "Genellikle rüzgarla birlikte etrafta uçan Elitriyanlar başlarının üzerinde yeterince alan olmadığında rahatsız olurlar.",
+  "origin.origins.elytrian.name": "Elytrian",
+  "origin.origins.elytrian.description": "Sık sık rüzgarlarda uçuşan Elytrianlar, kafalarının üzerinde yeterli alan bulunmadığında rahatsız olurlar.",
 
-  "origin.origins.enderian.name": "Enderiyan",
-  "origin.origins.enderian.description": "Ender Ejderhasının çocukları olarak doğan Enderiyanlar ışınlanma yeteneğine sahipler ancak suya karşı hassaslar.",
+  "origin.origins.enderian.name": "Enderian",
+  "origin.origins.enderian.description": "Ender Ejderhasının çocukları olarak doğan Enderianlar ışınlanabilirler fakat suya hassaslardır.",
 
   "origin.origins.shulk.name": "Shulk",
-  "origin.origins.shulk.description": "Shulkerlarla kan bağları olan Shulk'ın vücutları koruyucu kabuk benzeri bir deri ile donatılmıştır.",
+  "origin.origins.shulk.description": "Shulkerlar ile akraba olan Shulkların bedenleri koruyucu bir kabuğumsu deriyle kaplanmış.",
 
-  "power.origins.water_breathing.name": "Solungaçlar",
-  "power.origins.water_breathing.description": "Suyun altında nefes alabilirsin ama karada alamazsın.",
+  "power.origins.water_breathing.name": "Solungaç",
+  "power.origins.water_breathing.description": "Su altında nefes alabilirsin ama karada alamazsın.",
 
   "power.origins.aqua_affinity.name": "Su Adaptasyonu",
-  "power.origins.aqua_affinity.description": "Diğerlerinin karada yaptığı gibi su altında blokları kırabilirsiniz.",
+  "power.origins.aqua_affinity.description": "Diğerlerinin karada kırdığı hızda suyun altında blok kırabilirsiniz.",
 
   "power.origins.water_vision.name": "Islak Gözler",
-  "power.origins.water_vision.description": "Su altı görüşün mükemmel.",
+  "power.origins.water_vision.description": "Su altı görüşünüz mükemmel.",
 
   "power.origins.swim_speed.name": "Yüzgeçler",
-  "power.origins.swim_speed.description": "Su altında daha hızlısın.",
+  "power.origins.swim_speed.description": "Su altı hızınız arttırılmış.",
 
   "power.origins.like_water.name": "Su Gibi",
-  "power.origins.like_water.description": "Suyun altındayken istemediğiniz sürece dibe batmazsınız.",
+  "power.origins.like_water.description": "Su altındayken istemediğiniz halde batmazsınız.",
 
   "power.origins.fragile.name": "Narin",
-  "power.origins.fragile.description": "İnsanlardan 3 kalp daha az canınız var.",
+  "power.origins.fragile.description": "İnsanlara göre 3 daha az kalbiniz var.",
 
-  "power.origins.webbing.name": "Ağlama",
```

Comment:
Literal translation of "webbing" but it also means "Don't Cry". So just made it "web"